tools/src/main/java/org/apache/kafka/trogdor/common/WorkerUtils.java

```diff
@@ -128,12 +129,38 @@ public static void createTopics(
         Logger log, String bootstrapServers, Map<String, String> commonClientConf,
         Map<String, String> adminClientConf,
         Map<String, NewTopic> topics, boolean failOnExisting) throws Throwable {
+        // This method allows backward compatibility with the original signature of createTopics.
```

Comment:
We don't need backwards compatibility with the original signature of createTopics.  It's not a public API

Reply:
Ok, I'll remove it then.  I was nervous that this change could potentially break forks unknown to me.